### PR TITLE
Feat/layout

### PR DESCRIPTION
**Pull request by nikoaaman**
Added layout to main page. Added multiple images and assets. Styled the landing page

### CODE REVIEW

**Comment by GomezAgustin1** [comment]
Hi @nikoaaman can you leave as a comment the link to the figma?

File: style.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+nav ul li{    
+    display: inline;
+    list-style-type: none;
+    text-decoration: none;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 37px;
+    color: #ffffff;
+    margin: 5px;
```
Comment (GomezAgustin1) [inline style.css:31]: you can add to make the background transparent and fix your buttons
```
nav ul li button {
  background-color: transparent;
  border: none;
  cursor: pointer;
}
```
Eventhough I recommend using classes with relevant names to style your elements, to make it more readable

File: style.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    width: 1045px;
+    font-weight: 500;
+    font-size:30px;
+    line-height: 36.57px;
+}
+.play {
+    display: flex;
+    flex-direction: row;
+    color: #ffffff;
+    margin-left: 50px;
```
Comment (GomezAgustin1) [inline style.css:86]: Add this to remove the background to the play button
```
.play-button {
  background-color: transparent;
  border: none;
  cursor: pointer;
}
```

File: style.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    line-height: 34px;
+    padding: 0px 20px;
+}
+.info-button{
+    font-style: normal;
+    font-weight: 600;
+    font-size: 28px;
+    line-height: 34px;
+    border-radius: 25px;
+
```
Comment (GomezAgustin1) [inline style.css]: Add this, to remove the background 
```
.play-button {
  background-color: transparent;
  border: none;
  cursor: pointer;
}
```

File: style.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    background: linear-gradient(98.76deg, #1C0E0E 24.39%, #1C1833 81.33%);
+    border-radius: 50px;
+}
+/* NAVBAR */
+nav{display: flex;
+    padding: 45px 150px;
+    margin-left: 450px;
+}
+nav #marvel-logo{
+    width: 150px;
```
Comment (GomezAgustin1) [inline style.css:18]: I would recommend you to use the font-sizes with rems

File: style.css [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+*{
+    font-family: 'Montserrat', sans-serif;
+}
+body{
+    background-color: #000000;
+}
+.card-area{    
+    background: linear-gradient(98.76deg, #1C0E0E 24.39%, #1C1833 81.33%);
+    border-radius: 50px;
+}
```
Comment (GomezAgustin1) [inline style.css:11]: You can add a `.search-icon` class and `border-left: 2px solid white` to add the left line to it

File: index.html [hunk 6]
```diff
@@ -5,33 +5,45 @@
...
             <ul>
-                <li><a href=""></a>Home</li>
-                <li><a href=""></a>About</li>
-                <li><a href=""></a>Content</li>
-                <li><a href=""></a>Contact</li>
+                <li><a href="#home"></a>HOME</li>
+                <li><a href=""></a>COMICS</li>
+                <li><a href=""></a>GAMES</li>
+                <li><a href=""></a>NEWS</li>
+                <li><button><img src="/images/search.svg" alt="searc-icon"></button></li>
```
Comment (GomezAgustin1) [inline index.html]: Try to increase this image size a bit

**Review by GomezAgustin1 — CHANGES_REQUESTED** [review]

**Comment by nikoaaman** [comment]
> Hi @nikoaaman can you leave as a comment the link to the figma?

https://www.figma.com/file/oeP9YgGbcwZp5b9nTRkvoC/MARVELS-(Community)?type=design&node-id=3-2&t=M0jvjI0NWSCyCFzQ-0

File: style.css (hunk 1 above)
Comment (nikoaaman) [reply style.css:31]: Added the recommended changes. Added class "navButton" to both relevant buttons.

File: style.css (hunk 2 above)
Comment (nikoaaman) [reply style.css:86]: Added the proposed changes. Changed the class name from play-button to "playButton"

File: style.css (hunk 3 above)
Comment (nikoaaman) [reply style.css]: Same as previous comment. Mark as resolved.

File: style.css (hunk 4 above)
Comment (nikoaaman) [reply style.css:18]: Changed the lines 28 and 75 from 'font-size:30px;' to 'font-size:1.875rem;'. 
Likewise, changed lines 96 and 103 from 'font-size: 28px;' to 'font-size: 1.75rem;' and the line

File: style.css (hunk 5 above)
Comment (nikoaaman) [reply style.css:11]: Added the requested changes. 
Removed
nav div {
    border-left: solid #ffffff;
}

File: index.html (hunk 6 above)
Comment (nikoaaman) [reply index.html]: Increased the size of the search button from 36.67px to 55px. 
Increased the size of the user Icon from the same size to 70px.

**Pull request merged**